src/components/AdminDetails.vue

```diff
@@ -0,0 +1,156 @@
+        </div>
+      </div>
+      <div class="flex pt-6">
+        <div class="w-1/4 font-circular font-bold">Name</div>
+        <!-- <div class="w-3/4">--</div> -->
+        <div class="w-3/4">{{admin.firstName}} {{admin.lastName}}</div>
+      </div>
+      <div class="flex py-6 border-b border-gray-db">
+        <div class="w-1/4 font-circular font-bold">Email</div>
+        <div class="w-3/4">{{admin.email}}</div>
+      </div>
+      <div class="flex pt-6">
+        <div class="w-1/3 font-circular text-sm">Account created</div>
+        <div class="w-2/3 font-circular text-sm text-right">{{admin.joinedAt | dateTimeFormat}}</div>
+      </div>
+      <div class="flex pt-6">
+        <div class="w-1/3 font-circular text-sm">Created by</div>
+        <div class="w-2/3 font-circular text-sm text-right">--</div>
+      </div>
+      <div class="flex py-6 border-b border-gray-db">
+        <div class="w-1/3 font-circular text-sm">Last active</div>
+        <div class="w-2/3 font-circular text-sm text-right">--</div>
+      </div>
+      <div class="pt-6">
+        <div class="text-sm font-circular font-bold">Privileges</div>
+        <div class="h-64 overflow-y-scroll">
+          <label :for="permissionKey" v-for="(permissionObject, permissionKey) of admin.permissions" :key="permissionKey" class="border-b border-gray-300 relative block permission">
+            <input type="checkbox" :name="permissionKey" :id="permissionKey" class="invisible absolute">
+            <header class="py-2 px-1 flex justify-between items-center cursor-pointer capitalize">
+              <span>{{permissionKey | camelCaseSeperator}}</span>
+              <img src="./../assets/img/dropdown-indicator.svg" alt="dropdown indicator">
+            </header>
+            <div class="permission-items">
+              <div
+                @click.prevent="togglePermission(`${permissionKey}.${action}`)"
+                v-for="(value, action) of permissionObject"
+                :key="`${permissionKey}_${action}`"
+                :class="{'selected': value}"
+                class="permission-item flex items-center justify-between h-10 pl-3 cursor-pointer capitalize">
+                {{action}}
+              </div>
+            </div>
+          </label>
+        </div>
+      </div>
+      <div class="flex pt-16 actions">
+        <button class="relative border-black border w-40 py-2 px-3 flex justify-center disabled:font-gray-96 disabled:border-gray-96 items-center font-circular mr-4" :disabled="admin.isBlocked">Block</button>
+        <button class="relative border-black border w-40 py-2 px-3 flex justify-center disabled:font-gray-96 disabled:border-gray-96 items-center font-circular mr-4" :disabled="admin.isDeleted">Delete</button>
+      </div>
+    </div>
+    <div v-else class="flex items-center justify-center text-center text-2xl text-gray-c4 mt-20">
+      Select an Admin from the right
```

Comment:
I think this should be "Select an Admin from the left".

Reply:
yup will change it